parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
Yeah, I'd just add it to where the extensions are being built.

Regarding the path, you can interrogate the path itself via python, or use an if/else based on the arch. But I'd make sure the docs are correct before going to that effort.